src/MoBi.Presentation/Presenter/Main/MenuAndToolBarPresenter.cs

```diff
@@ -173,7 +173,7 @@ private void enableDefaultItems()
          _menuBarItemRepository[MenuBarItemIds.NotificationView].Enabled = true;
          _menuBarItemRepository[MenuBarItemIds.ComparisonView].Enabled = true;
          _menuBarItemRepository[MenuBarItemIds.OpenSimulation].Enabled = true;
-         _menuBarItemRepository[MenuBarItemIds.BuildingBlockExplorerView].Enabled = true;
```

Comment:
Are you actually removing the building block explorer? Mght be good to keep it around for now so that we can see what happens when loading older projects

Reply:
I have them both still, but I expect that before release, we will remove the bb explorer.